nuget/Auth0.OidcClient.Core.nuspec

```diff
@@ -101,11 +104,19 @@
         <dependency id="System.Text.Encodings.Web" version="5.0.1" />
         <dependency id="System.Text.Json" version="5.0.1" />
       </group>
+      <group targetFramework="net6.0">
+        <dependency id="IdentityModel.OidcClient" version="5.2.1" />
+        <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.12.2" />
+        <dependency id="System.Text.Encodings.Web" version="5.0.1" />
+        <dependency id="System.Text.Json" version="5.0.1" />
```

Comment:
These two dependencies do not mention to support .NET6 from the looks of it. Does this work when you try installing the nuget package in .NET6?

https://www.nuget.org/packages/System.Text.Json/5.0.1
https://www.nuget.org/packages/System.Text.Encodings.Web/5.0.1

Reply:
It's working when installing with .NET6, but I'm going to investigate it now.

Reply:
That's right, these two components are not necessary